README.md

```diff
@@ -22,7 +22,8 @@ See the [upgrade notes](https://github.com/greena13/react-hotkeys/releases/tag/v
 - [Browser key names](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values) and [Mousetrap syntax](https://github.com/ccampbell/mousetrap)
 - Define [global](#GlobalHotKeys-component) and [in-focus](#HotKeys-component) hot keys
 - [Display a list of available hot keys to the user](#Displaying-a-list-of-available-hot-keys)
-- [Defining custom key codes](#defining-custom-key-codes) for WebOS and other environments
+- [Define custom key codes](#defining-custom-key-codes) for WebOS and other environments
```

Comment:
🐥Typo spotted while doing this development - not actually related to intended functionality change.